assets/data/peopleParts/drauvenHelmet01.json

```diff
@@ -0,0 +1,63 @@
+{
+	"type": "extra",
+	"extraSlot": "zzz_drauven05_Helmet",
+	"requireAspects": ["drauvenSkin_naturalHead"],
+
+	"groups": [
+		{"groupIndex": 0, "male": true, "maleWeight": 1, "female": true, "femaleWeight": 1},
+		{"groupIndex": 1, "male": true, "maleWeight": 1, "female": true, "femaleWeight": 1},
+		{"groupIndex": 2, "male": true, "maleWeight": 1, "female": true, "femaleWeight": 1},
+		{"groupIndex": 3, "male": true, "maleWeight": 1, "female": true, "femaleWeight": 1}
+	],
+  "layers": [
+		{
+			"name": "replaceHead_drauvDetail_helmet1",
```

Comment:
🤔 It might be better to split each helmet off to its own file and use `requireAspects` to limit it to the appropriate head - this way we could possibly have more helmets in the future, and it would also make the preview show the appropriate helmet rather than all the helmets stacked on top of each other.

Reply:
the preview should show the correct helmet or if that's too much could we make a customization option that is "helmet" "yes", "no"  and cross the bridge of separating helmets out if/when there are multiple helmets to the head.

Reply:
Aye, the game seems to display all head layers at once (not just the ones that will appear based on the character's other aspects). I let Patrick know about this a while ago and he said he'd look into it, but assuming that it's not very high on their priority list it's pretty easy for us to just split this into multiple files (1/helmet)